backend/cloud/app.js

```diff
@@ -63,6 +65,27 @@ app.get("/t/:slug", async function inner(req, res, next) {
 app.get("/u/:id", makeWithOG(User, []));
 app.get("/a/:id", makeWithOG(Activity, ["objects", "team"]));
 
+app.get("/app-redir", (req, res) => {
+  const hostname = req.hostname
+  const pageConfig = Object.values(config.pages).find((page) =>
+    page.templateParameters.settings.PARSE_URL.includes(hostname)
+  )
+  const pageSettings = pageConfig.templateParameters.settings

```

Comment:
Let's just change the format for configuration to be more explicit, rather than matching against the PARSE_URL...

```
config = {
"app.wir.md": {..},
"app.affinity.wtf": {}
}
```

Reply:
Is it "app.affinity.wtf" or "community.affinity.wtf"?

Reply:
was just an example out off the top of my head. the url should be `community.affinity.wtf` I think...